articles/how-to-prepare-for-coding-interviews/index.md

```diff
@@ -0,0 +1,45 @@
+
+Whether you are a student or a working professional, effective preparation for software engineer interviews is essential if you want a job at a good company. *This tutorial covers everything you need to know to prepare for coding interviews in exceptional tech companies*.
+
+### Table of contents
+
+1.  [The interview process](#the-interview-process)
+2.  [The first round](#the-first-round)
+3.  [Technical interview](#technical-interview)
+4.  [HR interview](#hr-interview)
+5.  [Conclusion](#conclusion)
+
+### The interview process
+Reputed tech companies have a rigorous interview process to select the right candidate for the job. The first round is usually an online coding test conducted on platforms such as [Hackerrank](https://www.hackerrank.com/), [HackerEarth](https://www.hackerearth.com/), [Mettl](https://mettl.com/), etc. It consists of coding questions or a combination of coding questions and [MCQs](https://www.indiabix.com/online-test/aptitude-test/)(Multiple Choice Questions). Once the candidate clears the first round, they schedule a [telephonic interview](https://www.monsterindia.com/career-advice/phone-interview-tips-10-keys-to-succeed-in-a-telephonic-interview-6485.html) or an [onsite interview](https://www.algrim.co/302-onsite-interview) (The candidate is requested to go to the company office and attend a series of interviews). These interviews can be classified as technical or HR interviews. In technical interviews, you are asked questions about your resume, work experience(If any), fundamentals of computer science, Algorithms and Data structures, etc. HR interviews test your behavior, your ability to work well in a team, and whether you will be able to thrive in the company.
+
+### The first round
+The first round is usually an online coding round that tests the candidate's ability to use Algorithms and Data Structures to solve complex problems. This round tests your ability to understand and solve a given problem in a short amount of time. To ace this round, you must practice coding questions on platforms such as [Codechef](https://www.codechef.com/), Hackerrank, HackerEarth, etc. [This](https://www.section.io/engineering-education/how-to-start-competitive-programming/) is a good article on competitive coding and gives you a complete roadmap on how you can get started with it.
+
+### Technical interview
+Once you clear the first round, the recruiter will schedule face-to-face interviews. The first and most common question the interviewer will ask is to introduce yourself. Prepare a short introduction for yourself well in advance so that it becomes easier to answer the question in the interview. [This](https://www.themuse.com/advice/tell-me-about-yourself-interview-question-answer-examples) article covers this question in depth.
```

Comment:
Give a few pointers to prepare the short introduction for the interview. Mention a few points from the article that you have given the link to. This is to ensure maximum benefit to the reader.